hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/operations/Backup.java

```diff
@@ -161,12 +170,10 @@ public boolean validatesTarget() {
     public void onExecutionFailure(Throwable e) {
         if (backupOp != null) {
```

Comment:
What worries me is, if we have more than 1 backup, means we are operating by using `backupOpData`, the exception will not be print as in 1 backup-ed case. Here this null check can only be passed if we have 1 backup. 1 and more backup-ed cases are not parallel any more from this exception printing perspective.